docs/cross-refs.txt

```diff
@@ -0,0 +1,101 @@
+[`add_conditional_edges`][langgraph.graph.StateGraph.add_conditional_edges]
```

Comment:
Did not meant o commit this. This will be used to seed the contents of `link_map`